starknet-rpc-test/src/utils.rs

```diff
@@ -87,6 +87,14 @@ pub trait AccountActions {
         nonce: Option<u64>,
     ) -> TransactionExecution;
 
+    fn invoke_contract(
```

Comment:
Let's call invoke_contract inside transfer_tokens. All the code in invoke_contract is duplicated in transfer_tokens.

Reply:
done